docs/source/blocks/blockStyleWrapper.md

```diff
@@ -53,10 +56,10 @@ The `style` field is mapped to an `objectWidget`, and the `stylesSchema` adds th
 }
```

Comment:
I just noticed that this block is missing one `}`, which is why GitHub highlights the closing three backticks and the rest of the content in red.

Also the second to last `}` needs to be indented properly.

Reply:
Fixed.